openquake/calculators/scenario_damage.py

```diff
@@ -200,11 +200,19 @@ def post_execute(self, result):
         events_per_asset = (indices[:, 1] - indices[:, 0]).mean()
         logging.info('Found ~%d dmg distributions per asset', events_per_asset)
 
+        # avg_factor = factor used when computing the averages
+        oq = self.oqparam
+        if oq.investigation_time is None:  # scenario_damage
```

Comment:
If the job file for a scenario_damage calculation includes the `investigation_time` inadvertently, do we ignore it when reading the oq params?

Reply:
I would raise an error in that case, what do you think?

Reply:
That should work

> I would raise an error in that case, what do you think?

That should work

Reply:
Done